tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -435,4 +435,105 @@ virtual void TearDown() {
         EXPECT_EQ(0, verAdvances[i].height); // Yes actually!
     }
 #endif
+}
+
+TEST(CTFont, GetBoundingBox) {
+// Font names differ slightly between platforms
+// Windows may have slightly different versions of the same fonts compared to the reference platform
+#if TARGET_OS_WIN32
+    CFStringRef name = CFSTR("Arial Italic");
```

Comment:
>Arial Italic [](start = 30, length = 12)

better yet, have a test header file that defines these for OSX and Windows. this will reduce the #if defs and make it easier to maintain.